pkg/kubectl/cmd/auth/cani.go

```diff
@@ -238,3 +254,51 @@ func (o *CanIOptions) resourceFor(mapper meta.RESTMapper, resourceArg string) sc
 
 	return gvr
 }
+
+func (o *CanIOptions) runRulesCheck() (bool, error) {
+	result, err := o.SelfSubjectRulesClient.SelfSubjectRulesReviews().Create(
+		&authorizationapi.SelfSubjectRulesReview{
+			Spec: authorizationapi.SelfSubjectRulesReviewSpec{
+				Namespace: o.Namespace,
+			},
+		},
+	)
+	if err != nil {
+		return false, err
+	}
+
+	table := &metav1alpha1.Table{}
+	table.ColumnDefinitions = []metav1alpha1.TableColumnDefinition{
+		{Name: "VERBS", Type: "string"},
+		{Name: "NON-RESOURCE URLS", Type: "string"},
+		{Name: "RESOURCE NAMES", Type: "string"},
+		{Name: "API GROUPS", Type: "string"},
+		{Name: "RESOURCES", Type: "string"},
+	}
+
+	for _, rule := range result.Status.ResourceRules {
+		table.Rows = append(table.Rows, metav1alpha1.TableRow{
+			Cells: []interface{}{
+				rule.Verbs, "", rule.ResourceNames, rule.APIGroups, rule.Resources,
+			},
+		})
+	}
+	for _, rule := range result.Status.NonResourceRules {
+		table.Rows = append(table.Rows, metav1alpha1.TableRow{
+			Cells: []interface{}{
+				rule.Verbs, rule.NonResourceURLs, "", "", "",
+			},
+		})
+	}
+
+	printers.PrintTable(table, o.Out, printers.PrintOptions{})
```

Comment:
@fabianofranz I'm pretty sure this isn't printing right.

Reply:
cc @kubernetes/sig-cli-pr-reviews can anyone comment if this is correct?